Common/AR/source/Android/ArCorePointCloudRenderer.cpp

```diff
@@ -27,17 +27,13 @@ using namespace Esri::ArcGISRuntime::Toolkit::Internal;
   - https://github.com/google-ar/arcore-android-sdk/blob/master/samples/hello_ar_c/app/src/main/cpp/hello_ar_application.cc
   - https://github.com/google-ar/arcore-android-sdk/blob/master/samples/hello_ar_c/app/src/main/cpp/plane_renderer.cc
   */
-
-/*!
-  \internal
-  */
 ArCorePointCloudRenderer::ArCorePointCloudRenderer(ArCoreWrapper* arCoreWrapper) :
   m_arCoreWrapper(arCoreWrapper)
 {
   Q_CHECK_PTR(m_arCoreWrapper);
 }
 
-/*!
+/*
```

Comment:
Don't we still need the `/*!` for the doc to recognize this as something to ignore? This comment applies throughout.

Reply:
`ArCorePointCloudRenderer` is an internal class, no doc should be generated for this class. I don't know if it's better in this case to not provide doc at all? (I added doc with "\internal")

Reply:
I'll defer to @tdunn. I'm fine either way.